modules/nw-disabling-hsts.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id="nw-disabling-hsts_{context}"]
+= Disabling HTTP strict transport security per-domain or globally
+
+To disable HTTP strict transport security (HSTS) per-route or globally, you can set the `max-age` value in the route annotation to `0`.
+
+.Prerequisites
+
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Edit the Ingress conifg file:
```

Comment:
To edit the route you would edit the route config file or use `oc annotate`, e.g. `$ oc annotate route routename -n namespacename --overwrite=true "haproxy.router.openshift.io/hsts_header"="max-age=0"`